### ISSUE TITLE
Error when initializing Cordova Project without CLI

### ISSUE THREAD

**Issue by rohitb4** [driftyco/ionic-cli#950]
Error: Cannot find module 'core-util-is'
    at Function.Module._resolveFilename (module.js:337:15)
    at Function.Module._load (module.js:287:25)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (C:\Users\boggarap\AppData\Roaming\npm\node_modules\ionic\node_modules\ionic-app-lib\node_modules\cordova-lib\node_modules\npm\node_modules\request\node_modules\bl\node_modules\readable-stream\lib\_stream_duplex.js:26:12)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)

I'm trying to start an ionic app, have installed cordova globally as well.

**Comment by jthoms1** [comment]
Thanks for opening the issue. Could you provide us with additional information about your current setup by running `ionic info` within the project directory and pasting it into this issue.

**Comment by rohitb4** [comment]
`ionic info` output:

WARN: ionic.config.js has been deprecated, you can remove it.

Your system information:

Cordova CLI: Not installed
Ionic Framework Version: 2.0.0-beta.4
Ionic CLI Version: 2.0.0-beta.25
Ionic App Lib Version: 2.0.0-beta.15
OS:
Node Version: v4.2.3


******************************************************
 Dependency warning - for the CLI to run correctly,
 it is highly recommended to install/upgrade the following:

 Please install your Cordova CLI to version  >=4.2.0 `npm install -g cordova`

******************************************************

**Comment by jgw96** [comment]
Hello @rohitb4 ! Thanks for opening an issue with us! Would you mind running `cordova --version` and posting the output here?

**Comment by jgw96** [comment]
Hello all! As it seems it has been a while since there was any activity on this issue i will be closing it for now. Feel free to comment if you are still running into this issue. Thanks for using Ionic!

**Comment by jaypalkadam** [comment]
Hey guys ,

i have same issue like Initializing "cordova project without CLI" even i had installed the latest version of the cordova pls anybody help me to figurout this issue

**Comment by finessed** [comment]
Just had the same issue on a mac, was getting the ENOENT error. What I did was create my root user account. Went into my admin user account. Used su- root and input my password. Then I noticed that a bin/cordova was showing up in the error so I did a sudo npm uninstall -g cordova. Then did a sudo npm install -g cordova as root#. Hope this helps anyone with the same issues. I still find that only in root does cordova cli show in ionic info though. Still figuring that part out but ionic serve now works.
